src/bridges/erc4626/VaultBridge.sol

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright 2020 Spilsbury Holdings Ltd
```

Comment:
Please replace license and copyright with:

// SPDX-License-Identifier: Apache-2.0
// Copyright 2022 Aztec.